.travis.yml

```diff
@@ -7,6 +7,19 @@ scala:
 git:
   depth: false
 
+env:
+  matrix:
+    - TRAVIS_JDK=adopt@1.8.202-08
+    - TRAVIS_JDK=adopt@1.11.0-2
+
+matrix:
+  fast_finish: true
+  allow_failures:
+    - env: TRAVIS_JDK=adopt@1.11.0-2 # not fully supported but allows problem discovery
```

Comment:
Should we leave open #530 to fix these? Or introduce another ticket?

Reply:
I don't mind.  Which do you prefer?

Reply:
No preference - let's keep #530 open for this then?